modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Changing applied managed cluster CRs using policies
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
```

Comment:
You use both _content_ and _field_. Are these the same objects? Should you use the same term throughout?

Reply:
It really depends on what the user is removing from the CR, it can be a field, it can be several fields, depending on what they are trying to do. To make it more generic, I rephrased it a bit and replaced field with content to make it.

The procedure that we have here is just an example to guide the user through how to remove content from CRs.

Reply:
Also, changed mode to configuration, hopefully that clarifies it.